Src/WitsmlExplorer.Api/Services/BhaRunService.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Serilog;
+using WitsmlExplorer.Api.Query;
+using Witsml.ServiceReference;
+using WitsmlExplorer.Api.Models;
+using WitsmlExplorer.Api.Models.Measure;
+
+namespace WitsmlExplorer.Api.Services
+{
+    public interface IBhaRunService
+    {
+        Task<BhaRun> GetBhaRun(string wellUid, string wellboreUid, string BhaRunUid);
+    }
+
+    public class BhaRunService : WitsmlService, IBhaRunService
+    {
+        public BhaRunService(IWitsmlClientProvider witsmlClientProvider) : base(witsmlClientProvider) { }
```

Comment:
Ref comment in Routes, BhaRunService needs a method that retrieves all bhaRuns for a given wellbore